pyoptsparse/pyOpt_optimization.py

```diff
@@ -1698,13 +1699,17 @@ def __str__(self):
                             # Active upper bound
                             status += "u"
 
-                    text += fmt.format(
-                        idx, c.name, typ, lower, value, upper, status, lambdaStar[con_name][j], width=num_c
-                    )
+                    if not (minimal_print and not status):
+                        text += fmt.format(
+                            idx, c.name, typ, lower, value, upper, status, lambdaStar[con_name][j], width=num_c
+                        )
                     idx += 1
 
         return text
 
+    def __str__(self, minimal_print=False):
```

Comment:
Typically these dunder methods do not accept args/kwargs. I would just suggest to call `summary_str()` with `minimal_print=False` as the "canonical" to_string method.